pkg/runner/cluster_k8s.go

```diff
@@ -204,6 +204,33 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 
 	var gg errgroup.Group
 
+	// TODO: hack - this is the daemon, and not the user's local machine
```

Comment:
Put this in S3?

Reply:
We could, however as far as I understood @raulk , we want the `structured output` from tests to be read and returned right away to users. Not sure what is the point of storing this on S3, and then adding more logic to process it on `testground` by reading from S3.

S3 makes sense for assets, which are not structured, and will potentially be much bigger.

Reply:
There are different kinds of "structured output". We have stdout which is status messages, metrics etc. Then we have _logs_. Collecting all those logs and streaming them back to the user is likely going to overwhelm something somewhere.